python/flask/secure-static-file-serve.py

```diff
@@ -6,3 +6,7 @@
 def download_file(filename):
   # ruleid:avoid_send_file_without_path_sanitization
```

Comment:
You will need to disable this or add decorator

Reply:
@ulziibay there is a decorator -- just expand the diff :)